### PR TITLE
[27.x] Backports and rc1

### PR DESCRIPTION
**Pull request by fanquake**
Backports:
* https://github.com/bitcoin-core/gui/pull/812
* https://github.com/bitcoin-core/gui/pull/813
* https://github.com/bitcoin/bitcoin/pull/30085
* https://github.com/bitcoin/bitcoin/pull/30094
* https://github.com/bitcoin/bitcoin/pull/30097
* https://github.com/bitcoin/bitcoin/pull/30149
* https://github.com/bitcoin/bitcoin/pull/30151

Bump to 27.1rc1.

### CODE REVIEW

**Review by hebasto — APPROVED** [review]
ACK 9867e72b995e4a838c8e8e0feb159a9bdee8b76f, I have reproduced backporting locally.

**Review by willcl-ark — APPROVED** [review]
ACK cb6def3855427b613357696ba16a431c7964dbcc

Reviewed that all backports were unmodified, and that associated PR #'s match.

Lightly reviewed release notes and manpage and they look good to me.

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
-Bitcoin Core version 27.x is now available from:
+27.1rc1 Release Notes
+=====================
 
-  <https://bitcoincore.org/bin/bitcoin-core-27.x/>
+Bitcoin Core version 27.1 is now available from:
+
+  <https://bitcoincore.org/bin/bitcoin-core-27.1/>
```
Comment (stickies-v) [inline doc/release-notes.md:6]:
```suggestion
Bitcoin Core version 27.1rc1 is now available from:

  <https://bitcoincore.org/bin/bitcoin-core-27.1/test.rc1/>
```

**Review by stickies-v — COMMENTED** [review]
ACK cb6def3855427b613357696ba16a431c7964dbcc modulo small release notes fix

All backport commits are clean, except for:
- 77b2321ca03dcbd5f77060510dc8a19e7f4fdfa2 backported from 21b8a14d37c19ce292d5529597e0d52338db48a9: version bumped in https://github.com/bitcoin/bitcoin/pull/29707/ - LGTM

No diff with my local manpage generation.

**Review by hebasto — APPROVED** [review]
ACK cb6def3855427b613357696ba16a431c7964dbcc.

Feel free to update translations with `python3 ../bitcoin-maintainer-tools/update-translations.py` or cherry-pick the https://github.com/hebasto/bitcoin/commit/c9c42cc5e9f5740c4700b18a28d809db53880db2 commit.

**Review by hebasto — APPROVED** [review]
re-ACK f30e23e4b35b52728ebaa1a63ec9a6feac51ebb4,

**Review by willcl-ark — APPROVED** [review]
reACK f30e23e4b35b52728ebaa1a63ec9a6feac51ebb4

Didn't review the newly-added individual translations, but apart from that changes LGTM.

File: doc/release-notes.md (hunk 1 above)
Comment (stickies-v) [reply doc/release-notes.md]: nit but text is not updated to "271.rc1" in latest force push (link is fine now)

**Review by stickies-v — APPROVED** [review]
re-ACK f30e23e4b35b52728ebaa1a63ec9a6feac51ebb4, I'm getting the same translation update results as in 80032d691702f1e5eccb27571066bfc9b2415742

**Comment by stickies-v** [comment]
re-ACK 22701a43464ab27ea83c8b49e2732ee647909a10

**Comment by willcl-ark** [comment]
reACK 22701a43464ab27ea83c8b49e2732ee647909a10

on range-diff https://github.com/bitcoin/bitcoin/compare/f30e23e4b35b52728ebaa1a63ec9a6feac51ebb4..22701a43464ab27ea83c8b49e2732ee647909a10

**Review by hebasto — APPROVED** [review]
re-ACK 22701a43464ab27ea83c8b49e2732ee647909a10.

**Pull request merged**